src/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilder.cs

```diff
@@ -739,27 +720,27 @@ private void SetGenParamCustomAttributeNoLock(CustAttr ca)
         #endregion
 
         #region Object Overrides
-        public override string ToString()
+        public override string? ToString()
```

Comment:
Does this ever return null?

Reply:
method called below return null:
`return TypeNameBuilder.ToString(this, TypeNameBuilder.Format.ToString);`

Reply:
> method called below return null

In what situation?

Reply:
```
internal static string? ToString(Type type, Format format)
        {
            if (format == Format.FullName || format == Format.AssemblyQualifiedName)
            {
                if (!type.IsGenericTypeDefinition && type.ContainsGenericParameters)
                    return null;
            }
```
I see now format parameter never allow return null, but still cannot remove '?' because of compiler issue https://github.com/dotnet/roslyn/issues/23268#issuecomment-482306719. Will add TODO-Nullable and link to the issue

Reply:
you can suppress the warning and and a TODO